### PR TITLE
[fix] toolbar initial position is incorrect

### PR DESCRIPTION
**Pull request by suli1**
Toolbar初始显示位置偏下，有时会被系统导航栏遮挡，显示不全

## Pull Request Checklist

- [x] I have read the [UME contribution document](../CONTRIBUTING.md) and understand how to contribute, commit the code according to the rules. 我已阅读过 UME 贡献文档，并了解如何进行贡献，按照规则提交了代码
- [x] I have added the necessary comments in code to ensure that other contributors can understand the reason for the change. 我在修改中已经添加了必要的注释，以确保便于其他贡献者理解修改原因
- [x] The code has been formatted by dartfmt before push. 代码在提交前已经经过 dartfmt 进行了格式化
- [x] Change does not involve the adjustment of test cases. Or all existing and new tests are passing. 改动不涉及测试用例调整，或 example 工程与单元测试已经完全跑通

If you need help, consider [Join ing the ByteDance Flutter Exchange Group](https://applink.feishu.cn/client/chat/chatter/add_by_link?link_token=67au2f75-3783-41b0-8868-0fc0178f1fd8).

如有任何问题，可以[加入字节跳动 Flutter 交流群](https://applink.feishu.cn/client/chat/chatter/add_by_link?link_token=67au2f75-3783-41b0-8868-0fc0178f1fd8)。

Or contact [author](mailto:sunkai.dev@bytedance.com).

或随时[联系开发者](mailto:sunkai.dev@bytedance.com)。

### CODE REVIEW

File: example/android/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -6,7 +6,7 @@
          additional functionality it is fine to subclass or reimplement
          FlutterApplication and put your custom class here. -->
     <application
-        android:name="io.flutter.app.FlutterApplication"
```
Comment (talisk) [inline example/android/app/src/main/AndroidManifest.xml]: It seems to have nothing to do with the function modification, should you delete it?

File: example/android/build.gradle [hunk 2]
```diff
@@ -1,5 +1,5 @@
 buildscript {
-    ext.kotlin_version = '1.3.50'
```
Comment (talisk) [inline example/android/build.gradle]: He seems to have nothing to do with the function modification, should you delete it

**Review by talisk — CHANGES_REQUESTED** [review]

**Comment by suli1** [comment]
Resolved

File: lib/core/ui/toolbar_widget.dart [hunk 3]
```diff
@@ -27,29 +27,26 @@ const double _dragBarHeight = 32;
...
     _dy += details.delta.dy;
-    _dy = min(max(0, _dy),
-        MediaQuery.of(context).size.height - _minimalHeight - _dragBarHeight);
+    _dy = min(max(0, _dy), _maxDy);
     setState(() {});
   }
 
   @override
   Widget build(BuildContext context) {
-    if (_windowSize.isEmpty) {
```
Comment (talisk) [inline lib/core/ui/toolbar_widget.dart:48]: In some case, `windowsSize` will be empty... I will test it and make sure it couldn't be empty. Please wait a while.

**Pull request merged**